src/main/kotlin/com/terraformation/backend/accelerator/db/ModuleEventStore.kt

```diff
@@ -214,4 +214,26 @@ class ModuleEventStore(
       else -> EventStatus.Ended
     }
   }
+
+  private fun eventProjectsMultiset(): Field<Set<ProjectId>> {
```

Comment:
This function was moved from the EventProjectsMultiset function since this can be made private now